### DISCUSSION TITLE
Is there a way to change keyboard and mouse that needs remap?

### DISCUSSION THREAD

**Discussion by roozye** [AntiMicroX/antimicrox#779] [General]
In case that I use multiple KbM and even the Virtual KbM to play games they need only KbM with a software called Nucleus Coop, while I may dont have extra KbM with me, and using that virtual drivers, Is there any way I change the default keyboard and mouse they need to remaped?

**Comment by mrbeauregarde** [comment]
https://github.com/AntiMicroX/antimicrox/wiki/Automatically-Loading-Profiles
